Author: Roland Rosenfeld <roland@debian.org>
Description: use gengbx.c instead of gengbx.o in SRCS, otherwise gccmakedep
 fails (Closes: #759236)
Bug-Debian: http://bugs.debian.org/759236

--- a/fig2dev/dev/Imakefile
+++ b/fig2dev/dev/Imakefile
@@ -30,7 +30,7 @@
 READXPMO=readxpm.o
 #endif
 
-SRCS =  genbox.c gencgm.c gendxf.c genepic.c gengbx.o genibmgl.c genlatex.c genmap.c genmf.c genpic.c \
+SRCS =  genbox.c gencgm.c gendxf.c genepic.c gengbx.c genibmgl.c genlatex.c genmap.c genmf.c genpic.c \
 	genpictex.c genps.c genpdf.c genpstex.c genpstricks.c gentextyl.c gentk.c genptk.c gentpic.c \
 	genbitmaps.c genge.c genmp.c genemf.c gensvg.c genshape.c setfigfont.c psencode.c \
 	readpics.c readeps.c readgif.c readpcx.c readppm.c readpng.c readxpm.c \
